cmd/car/get.go

```diff
@@ -176,7 +178,16 @@ func writeCarV2(ctx context.Context, rootCid cid.Cid, output string, bs *blockst
 		return err
 	}
 
-	err = traversalProgress.WalkMatching(rootNode, s, func(p traversal.Progress, n datamodel.Node) error { return nil })
+	err = traversalProgress.WalkMatching(rootNode, s, func(p traversal.Progress, n datamodel.Node) error {
+		lb, ok := n.(datamodel.LargeBytesNode)
+		if ok {
+			rs, err := lb.AsLargeBytes()
+			if err == nil {
+				io.Copy(io.Discard, rs)
```

Comment:
do we care about this error?

Reply:
the potential one from `io.Copy`, you mean?

Reply:
yup